multiapi-engine/src/test/java/com/sngular/api/generator/plugin/openapi/OpenApiGeneratorFixtures.java

```diff
@@ -755,15 +755,11 @@ static Function<Path, Boolean> validateAdditionalProperties() {
 
     final String DEFAULT_MODEL_API = "generated/com/sngular/multifileplugin/testadditionalproperties/model";
 
-    final String DEFAULT_EXCEPTION_API = "generated/com/sngular/multifileplugin/testadditionalproperties/model/exception";
-
-    final List<String> expectedTestApiFile = List.of("openapigenerator/testAdditionalProperties/assets/TestAPI.java");
+    final List<String> expectedTestApiFile = List.of("openapigenerator/testAdditionalProperties/assets/TestApi.java");
 
     final List<String> expectedTestApiModelFiles = List.of("openapigenerator/testAdditionalProperties/assets/TestDTO.java");
 
-    final List<String> expectedExceptionFiles = List.of();
-
-    return (path) -> commonTest(path, expectedTestApiFile, expectedTestApiModelFiles, DEFAULT_TARGET_API, DEFAULT_MODEL_API, expectedExceptionFiles, DEFAULT_EXCEPTION_API);
+    return (path) -> commonTest(path, expectedTestApiFile, expectedTestApiModelFiles, DEFAULT_TARGET_API, DEFAULT_MODEL_API, Collections.emptyList(), null);
```

Comment:
Este cambio es necesario?

Reply:
It's not necessary but the list is not needed